docs/release-notes/april-2024.md

```diff
@@ -0,0 +1,160 @@
+- Added the ability to copy autofix suggestions displayed on the **Findings** page.
+- Added the ability to filter findings generated by Pro rules on the **Findings** page.
+
+### Changed
+
+- Cross-file (intrafile) analysis is now the default for Semgrep Code.
+- Switched regex engines; some rules may need to be updated, since Semgrep's new regex engine is stricter.
+
+### Fixed
+
+- Fixed the SARIF output obtained from the CLI so that it includes dataflow traces.
+- Semgrep no longer uses the `LOG_LEVEL` and `PYTEST_LOG_LEVEL` environment variables to determine the log level. Semgrep only considers `SEMGREP_LOG_LEVEL`, as well as `PYTEST_SEMGREP_LOG_LEVEL` in Semgrep's Pytest tests.
+- Fixed the ecosystem used for Elixir from Mix to Hex.
+- Fixed an issue with interfile diff scans where the removal of pre-existing findings
+didn't work properly when adding a new file or renaming an existing file.
+- Fixed an issue where findings reopened after they were initially removed when the findings metadata was changed.
+- Fixed an issue where bulk triage did not work.
+- **IDE Extensions**: Semgrep waits longer for users to log in from the IDE.
+- **CLI**:
+  - Upon completion, `semgrep ci` sends a message to Semgrep AppSec Platform to mark the scan as completed.
+  - Fixed an issue where `semgrep ci --oss-only` crashed when Semgrep Secrets was enabled.
+
+## ⛓️  Supply Chain
+
+### Added
+
+- Added a new **Finding details** page. In Semgrep AppSec Platform, click on **Supply Chain > Details** on the specific finding's card. The finding details page displays an all triage activity, in-depth description of the vulnerability, remediation, and the rule pattern that generated the finding.  <!-- 13780 -->
+-
+
+### Changed
+
+<!-- should we include the new UI for SCA  -->
+
+- The Supply Chain page UX has been redesigned. Improvements include:
+    - You can now filter by project, dependency, and CVE.
+    - Updated reachability categories.
+    - Vulnerabilities are now grouped by the rule that detected them.
+    - Triage multiple vulnerabilities at once by clicking multiple <i class="fa-solid fa-square-check"></i> checkboxes. You can <kbd>Shift + click</kbd> to select a range.
+    - Within the projects page, the count of reachable vulnerabilities counts only Always reachable <!-- 13645 -->
+
+### Fixed
+
+- Fixed an issue where tooltips for conditionally reachable vulnerabilities were not being displayed. <!-- 13775 -->
+
+## 🤖 Semgrep Assistant
+
+### Changed
+
+- Assistant usage is now capped by an hourly rate rather than a monthly limit.
+
+### Fixed
+
+- Fixed an issue where Assistant sent PR or MR comments for Supply Chain and Secrets findings; Assistant should only be doing so for Code findings.
+
+## 🔐 Semgrep Secrets
+
+<!--
+For Katie's draft
+- Superseded rules
+- Historical scans
+- Custom rule-writing
+-->
+
+### Added
+
+- Added historical scanning, which is the ability to scan Git commits for leaked credentials.
```

Comment:
I just realized we mentioned historical scanning and superseded rules last month; should we bring it up again, or should I remove this?

Reply:
We can remove it. Looks to be a light month for external-facing changes.